src/mixins/dom.js

```diff
@@ -0,0 +1,34 @@
+// DomMixin
+//  ---------
+
+import Backbone from 'backbone';
+
+export default {
+  createBuffer() {
+    return document.createDocumentFragment();
+  },
+
+  appendChildren(el, children) {
+    Backbone.$(el).append(children);
+  },
+
+  beforeEl(el, sibling) {
+    Backbone.$(el).before(sibling);
+  },
+
+  replaceEl(newEl, oldEl) {
+    Backbone.$(oldEl).replaceWith(newEl);
+  },
+
+  detachContents(el) {
+    Backbone.$(el).contents().detach();
+  },
+
+  attachHTML(el, html) {
+    Backbone.$(el).html(html);
+  },
+
+  removeEl(el) {
+    Backbone.$(el).remove();
```

Comment:
the cost of [Backbone.$](https://github.com/jquery/jquery/blob/305f193aa57014dc7d8fa0739a3fefd47166cd44/src/core/init.js#L101) looks trivial.
 
I went with `Backbone.$(el).remove();` to give as much override opportunity as possible, in case VDOM diffing might treat removing differently.